test/release/examples/primers/forallLoops.chpl

```diff
@@ -33,7 +33,7 @@ an expression. Both kinds are shown in the following sections.
 --------------------------------
 
 In the following example, the forall loop iterates over the array indices
-in parallel:
+in parallel. Since the loop indirectly loops over ``A`` and the body modifies it, an explicit ``ref`` intent must be used.
```

Comment:
Here, too, drop " indirectly loops over ``A`` and the body", then replace "it" with "A". Also please wrap the line to ~78 columns.